### DISCUSSION TITLE
如何动态更新 interacting 的 nodeMovable 值？

### DISCUSSION THREAD

**Discussion by iPonyo** [antvis/X6#4348] [Q&A]
非编辑模式下，设置 nodeMovable 为 false，进入编辑模式时，如何动态更新该属性的值？
`const graph = new Graph({
      container: document.getElementById('graphContainer'),
      interacting: {
        nodeMovable: false
      }
 })`

**Comment by zlecheng** [comment]
重新初始化配置？
